extensions/rich_text_components/components_test.py

```diff
@@ -269,4 +292,7 @@ def test_component_e2e_tests(self):
                 text_inside_constant = text_inside_constant[
                     text_inside_constant.find(',') + 1:]
 
+        # Disabled for svgeditor, will be removed in second
```

Comment:
Please add a TODO that points to an issue on GitHub that is assigned to you.

Reply:
Done.